packages/react-charting/src/components/DonutChart/DonutChartRTL.test.tsx

```diff
@@ -0,0 +1,205 @@
+import { render, screen, queryAllByAttribute, fireEvent, act } from '@testing-library/react';
+import { chartPoints } from './DonutChart.test';
+import { DonutChart } from './index';
+import * as React from 'react';
+import { DarkTheme } from '@fluentui/theme-samples';
+import { ThemeProvider } from '@fluentui/react';
+import * as utils from '../../utilities/utilities';
+import '@testing-library/jest-dom';
```

Comment:
this is not a good pattern. to extend jest matchers you need to register it within `packages/react-charting/config/tests.js` 

```diff
/** Jest test setup file. */

const { setIconOptions } = require('@fluentui/react/lib/Styling');
const { configure } = require('enzyme');
const Adapter = require('@wojtekmaj/enzyme-adapter-react-17');

// Suppress icon warnings.
setIconOptions({
  disableWarnings: true,
});

// Configure enzyme.
configure({ adapter: new Adapter() });

+ require('@testing-library/jest-dom');

```

Reply:
based on offline convo, that doesn't work because jest-dom needs to be setup within `setupFilesAfterEnv`

So lets do following:

<img width="750" alt="image" src="https://user-images.githubusercontent.com/1223799/235121600-3976cbc9-cc75-4fc6-a59e-94baf8098ea5.png">

<img width="615" alt="image" src="https://user-images.githubusercontent.com/1223799/235121635-024ff0ff-7ee1-4502-9cbb-94384f634dfc.png">

Reply:
done